samples/bluetooth/iso_time_sync/src/cis_central.c

```diff
@@ -174,7 +171,15 @@ void cis_central_start(bool do_tx, uint8_t retransmission_number, uint16_t max_t
 
 	configured_for_tx = do_tx;
 	if (do_tx) {
-		iso_tx_init(retransmission_number);
+		/** scan_start is registered as a callback that is triggered when an
+		 * ISO channel connects to avoid a situation where the
+		 * LE HCI Create CIS command is called while there is a pending CIS
+		 * connection. The CIS central starts scanning for a new connection only after a
+		 * pending CIS connection has completed. This must be done because the
+		 * LE HCI Create CIS command is disallowed when there a CIS connection is
+		 * pending. See Core Specification Version 5.4, Vol 6, Part B, section 7.8.99.
+		 */
+		iso_tx_init(retransmission_number, &scan_start);
```

Comment:
If I'm not mistaken, then this will now also scan if there are no more channels, is this intended?